ecs-init/engine/engine.go

```diff
@@ -168,6 +176,8 @@ func (e *Engine) PreStartGPU() error {
 	envVariables := docker.LoadEnvVars()
```

Comment:
Do we want a debug line to print all env vars here? 
We have debug output for user overrides [here](https://github.com/aws/amazon-ecs-init/blob/b2584f1ee3446b767e7ffc37fadf88cdeeec2aee/ecs-init/docker/docker.go#L354), would be good to see the final version too

Reply:
discussed offline, unfortunately ecs-init doesn't currently have the ability to adjust logging level.